PendingReleaseNotes.md

```diff
@@ -1,17 +1,21 @@
 # Major Themes
 
 ## Action Required
+- The order in which the Pods should be updated is: MON, MGR, MDS, OSD, Rook API and then the Rook Operator.
```

Comment:
this comment says that the MGRs need to be first with this monitoring change: https://github.com/rook/rook/pull/1154#issuecomment-354342886

which is correct?  if you can't repro the crash during upgrade, maybe it's not worth mentioning it here actually.  It could have been a bug that was fixed with 12.2.2?

Reply:
Could be, I removed it.